admin_scripts/ubuntu/ubuntu_16.04/if_oracle_java_fails.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+dpkg --configure -a
+apt-get -f install -y
+apt-get update && sudo apt-get upgrade -y
```

Comment:
Is this still necessary? _System - Opdatér alt_ now tries to recover from broken installations.

(Quite apart from that, how much do customers depend on Oracle's Java VM these days?)